DQM/L1TMonitor/src/L1TStage2uGT.cc

```diff
@@ -38,6 +43,22 @@ void L1TStage2uGT::dqmBeginRun(edm::Run const& iRun, edm::EventSetup const& evtS
    if (!gtUtil_->getAlgBitFromName(algoNameLastBxInTrain_, algoBitLastBxInTrain_)) {
       edm::LogWarning("L1TStage2uGT") << "Algo \"" << algoNameLastBxInTrain_ << "\" not found in the trigger menu " << gtUtil_->gtTriggerMenuName() << ". Could not retrieve algo bit number.";
    }
+   for(unsigned int i=0;i<unprescaledAlgoShortList_.size();i++){
+     if (gtUtil_->getAlgBitFromName(unprescaledAlgoShortList_.at(i), algoBitUnpre_) && (algoBitUnpre_!=-1)) {
+        unprescaledAlgoBit_.push_back(algoBitUnpre_);
```

Comment:
If one or more algos in the short list are not found the vectors with the algo names and with the algo bits are not synchronised anymore with the index.
Consider filling a std::vector<std::pair<std::string, int>> with the existing algo names and algo bits and work with those for the rest of the code.